.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
```

Comment:
Did you generate this rubocop todo with `rubocop --auto-gen-config  --auto-gen-only-exclude --exclude-limit 10000`? or something like it?

Reply:
I used `rubocop --auto-gen-config --exclude-limit 10000` because that's what the comment in the previous TODO said was used, but I can re-run with `--auto-gen-only-exclude` if that's better.

Reply:
Ah, I see how `--auto-gen-only-exclude` would help here.  It would avoid having the todo add things like this:

```
Metrics/AbcSize:
  Exclude:	  Max: 51
```

I'll do that now!

Reply:
Updated!